src/web/libdoc/i18n/translations.json

```diff
@@ -56,5 +56,34 @@
     "generatedBy": "Luotu",
     "on": "",
     "chooseLanguage": "Valitse kieli"
+  },
+  "nl": {
+    "code": "nl",
+    "intro": "Introductie",
+    "libVersion": "Bibliotheek versie",
+    "libScope": "Bibliotheek bereik",
+    "importing": "Importeren",
+    "arguments": "Parameters",
+    "doc": "Documentatie",
+    "keywords": "Actiewoorden",
+    "tags": "Labels",
+    "returnType": "Andwoord type",
+    "kwLink": "Actiewoord link",
+    "argName": "Benoemde parameters",
+    "varArgs": "Variabel aantal parameters",
+    "varNamedArgs": "Variable aantal benoemde parameters",
```

Comment:
Typo: Variabel. Al twijfel ik of je het in deze context een _variabel aantal_ wil noemen? Dit is de situatie waar je een `dict` meegeeft. Misschien: _set van benoemde parameters_? Ook hier zou ik het graag eerst in context zien.